Src/WitsmlExplorer.Frontend/components/Modals/ReportModal.tsx

```diff
@@ -60,10 +61,18 @@ export const ReportModal = (props: ReportModal): React.ReactElement => {
       confirmText="Ok"
       showCancelButton={false}
       content={
-        <>
+        <ContentLayout>
           {report ? (
-            <ContentLayout>
-              {report.summary && report.summary.includes("\n") ? (
+            <>
+              {report.warningMessage && (
+                <StyledBanner colors={colors}>
+                  <Banner.Icon variant="warning">
+                    <Icon name="warning_outlined" />
```

Comment:
Is it possible to center the icon in the circle, or just remove the circle?
![image](https://github.com/equinor/witsml-explorer/assets/70512270/c613ce6d-fe4d-4911-a3a8-1e0b092c49c8)

Reply:
I used the Equinor Banner component, and I believe this issue lies with them. While it is possible to change the background, don't you think it's better to stick with the default Equinor design?

Reply:
I think it's fine to change the background color. It would also make it look more like it does in an error. The icon can also possibly be changed to match that one.

![image](https://github.com/equinor/witsml-explorer/assets/70512270/313c65c4-1ea3-4ea9-9605-3bf28f11c4cf)